content/en/serverless/azure_app_services.md

```diff
@@ -65,20 +71,20 @@ The Datadog extension for Azure App Service provides additional monitoring capab
 
 ### Installation
 
-1. Configure the Azure integration to monitor your web app and verify it is configured correctly by ensuring that you see an `azure.app_service.count` metric tagged with the name of your web application. **Note**: This step is critical for metric/trace correlation, functional trace panel views in the Datadog portal, and accurate billing.
+1. Configure the [Azure integration][1] to monitor your web app or function. Verify it is configured correctly by ensuring that you see the corresponding `azure.app_service.count` or `azure.functions.count` metric in Datadog. **Note**: This step is critical for metric/trace correlation, functional trace panel views, and avoiding various broken user experiences on the Datadog site.
 
-2. Open the [Azure Portal][11] and navigate to the dashboard for the Azure App Service instance you wish to instrument with Datadog.
+2. Open the [Azure Portal][3] and navigate to the dashboard for the Azure app you wish to instrument with Datadog.
 
 3. Go to the Application settings tab of the Configuration page.
     {{< img src="infrastructure/serverless/azure_app_services/config.png" alt="configuration page" >}}
-4. Add your Datadog API key as an application setting called `DD_API_KEY` and a value of your [Datadog API Key][12].
+4. Add your Datadog API key as an application setting called `DD_API_KEY` and a value of your [Datadog API Key][4].
     {{< img src="infrastructure/serverless/azure_app_services/api_key.png" alt="api key page" >}}
 5. Configure optional application settings:
     - Set the `DD_SITE` to {{< region-param key="dd_site" code="true" >}} (defaults to `datadoghq.com`).
     - Set `DD_ENV` to group your traces and custom statistics.
     - Set `DD_SERVICE` to specify a service name (defaults to your web app name).
     - Set `DD_LOGS_INJECTION:true` for correlation with application logs from your web app.
```

Comment:
This 'DD_LOGS_INJECTION:true' is not required for Java since it is on by default.
It still needs to be present for .NET